### PR TITLE
Add Filmweb

### PR DESCRIPTION
**Pull request by NullPiotrException**
Filmweb is a Polish equivalent of IMDB so I also added it to Entertainment. It's also the 2nd biggest movie database in the world (after IMDB).

[Alexa stats](https://www.alexa.com/siteinfo/filmweb.pl)

I already created [PR](https://github.com/2factorauth/twofactorauth/pull/3062) for Filmweb in the past but it was rejected (sort of) because Filmweb didn't have a paid plan. Well, now it does. It's called [Filmweb+](https://www.filmweb.pl/plus).

### CODE REVIEW

File: _data/entertainment.yml [hunk 1]
```diff
@@ -27,6 +27,14 @@ websites:
       img: deezer.png
       tfa: No
 
+    - name: Filmweb
+      url: https://www.filmweb.pl
+      twitter: filmweb
+      facebook: filmweb
+      img: filmweb.pl
```
Comment (kmpoppe) [inline _data/entertainment.yml]: There's a typo here ;-)

Comment (kmpoppe) [reply _data/entertainment.yml]:
```suggestion
      img: filmweb.png
```

Comment (kmpoppe) [reply _data/entertainment.yml]: @einsteinsfool As you didn't allow changes to your PR from other members, you'd need to change this yourself :)
// Kai

Comment (NullPiotrException) [reply _data/entertainment.yml]: My bad, sorry. I guess I'm too used to writing the URL. Fixed. :-)

**Comment by Carlgo11** [comment]
@einsteinsfool what [site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#site-criteria) does the site meet?

**Comment by kmpoppe** [comment]
> @einsteinsfool what [site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#site-criteria) does the site meet?

@Carlgo11 I think Filmweb+ is a paid service, so Site Criteria 3 should be met, right?

**Comment by NullPiotrException** [comment]
@Carlgo11 kmpoppe is correct, as I mentioned in the last paragraph. I also linked to my last PR for Filmweb - #3062 where it's mentioned.

**Comment by Carlgo11** [comment]
I agree with what other maintainers have mentioned in #3062. It doesn't seem to meet the site criteria. At least not yet.

**Comment by NullPiotrException** [comment]
Have you readed my and kmpoppe's comments? The site criterion 3 is meeted. The site deals with money. The manual says "If one of the following criteria is met, it belongs on twofactorauth.org". And other maintainers mentioned if BEFORE Filmweb+ that users pay for (which I mentioned in my description of this PR).

Please reopen and merge.

**Review by kmpoppe — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**